how-to/how-to-load-kernel-modules-with-kata.md

```diff
@@ -0,0 +1,98 @@
+# Loading kernel modules
+
+A new feature for loading kernel modules was introduced in Kata Containers 1.9.
+The list of kernel modules and their parameters can be provided using the
+configuration file or OCI annotations. The [Kata runtime][1] gives that
+information to the [Kata Agent][2] through gRPC when the sandbox is created.
+The [Kata Agent][2] will insert the kernel modules using `modprobe`, hence
+modules dependencies are resolved automatically.
+
+The sandbox will not be started when:
+
+  * A kernel module is specified and the `modprobe` command is not installed in
+    the guest or it fails loading the module.
+  * The module is not available in the guest or it doesn't met the guest kernel
+    requirements, like architecture and version.
+
+In the following sections are documented the different ways that exist for
+loading kernel modules in Kata Containers.
+
+- [Using Kata Configuration file](#using-kata-configuration-file)
+- [Using annotations](#using-annotations)
+
+# Using Kata Configuration file
+
+The list of kernel modules and parameters can be set in the `kernel_modules`
+option as a coma separated list, where the first word of each element is
+considered as the module name and the rest as its parameters.
+
+In the following example, `e1000e` is the module name and the rest its
```

Comment:
> the rest

I'd actually describe this fully to avoid any confusion. Something like:

> The following example specifies two modules to load: `e1000e` and `i915`. Two parameters are specified for the `e1000` module: `InterruptThrottleRate` (which takes an array of integer values) and `EEE` (which requires a single integer value).